tests/models/resnet/test_modeling_resnet.py

```diff
@@ -141,9 +141,17 @@ class ResNetModelTest(ModelTesterMixin, unittest.TestCase):
     attention_mask and seq_length.
     """
 
-    all_model_classes = (ResNetModel, ResNetForImageClassification) if is_torch_available() else ()
+    all_model_classes = (
+        (
+            ResNetModel,
+            ResNetForImageClassification,
+            ResNetBackbone,
+        )
+        if is_torch_available()
+        else ()
+    )
 
-    fx_compatible = True
+    fx_compatible = False
```

Comment:
The problem is, the fx_compatible flag requires all model classes to be supported by torch.fx. Since ResNetBackbone is a new class, one would need to add support for it separately here: https://github.com/huggingface/transformers/blob/main/src/transformers/utils/fx.py.

So @michaelbenayoun agreed that I can set this flag to False and that we can add support for the backbone classes in a separate PR.

Reply:
Is this ok for you?

Reply:
AS mentioned in the internal thread, if all is needed is adding the backbone task in the mapping file, I would rather have it done in this PR.